crashreports/build.gradle

```diff
@@ -45,13 +45,15 @@ dependencies {
   implementation fileTree(include: ['*.jar'], dir: 'libs')
   implementation project(path: ':utils')
   api project(path: ':aptoide-analytics-default-implementation')
-  androidTestImplementation("com.android.support.test.espresso:espresso-core:2.2.2", {
+  androidTestImplementation("com.android.support.test.espresso:espresso-core:3.0.2", {
     exclude group: 'com.android.support', module: 'support-annotations'
   })
   testImplementation "junit:junit:4.12"
   implementation "io.reactivex:rxjava:${RXJAVA_VERSION}"
+  // implementation "io.reactivex.rxjava2:rxjava:${RXJAVA2_VERSION}"
   implementation "io.reactivex:rxandroid:${RXANDROID_VERSION}"
-  implementation "nl.littlerobots.rxlint:rxlint:${Rx_lint_version}"
+  implementation "io.reactivex.rxjava2:rxandroid:${RXANDROID_VERSION}"
```

Comment:
This will try to resolve io.reactivex.rxjava2:rxandroid:1.2.1 ; It won't compile.

Reply:
should be fixed now